buildscripts/update-lobster.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+set -o errexit
+set -o xtrace
+
+SCRIPTS_DIR=buildscripts
```

Comment:
If these are going to be relative to the evergreen home directory, they need to use EVGHOME.

All of these should be relative to EVGHOME.